src/components/project/dto/project.dto.ts

```diff
@@ -196,10 +197,16 @@ class Project extends Interfaces {
    * Optimization for {@link ProjectResolver.engagements}.
    * This doesn't account for changesets or item filters.
    */
-  @RequiredWhenNotInDev({
-    field: 'engagements',
-    isMissing: (project) => project.engagementTotal === 0,
-  })
+  @Disabled(`
+    I'm not convinced this wont have unintended consequences.
+    it is still handled with the workflow condition currently and deletes are
+    restricted, so this is a super edge case effectively.
```

Comment:
Proof.

    I'm not convinced this won't have unintended consequences.
    It is still handled with the workflow condition currently and deletes are
    restricted, so this is effectively a super edge case.